bin/critterget.py

```diff
@@ -104,26 +94,40 @@ def authpost (postdata='',keyget=''):
             handler=urllib2.HTTPSHandler(debuglevel=1)
         else :
             handler=urllib2.HTTPSHandler(debuglevel=0)
-
+            
         opener = urllib2.build_opener(handler)
         urllib2.install_opener(opener)
 
         resptext=urllib2.urlopen(request).read()
         data = json.loads(resptext)
-
+    
     except urllib2.URLError, e:
         print u'{} MessageType="CrittercismError" Crittercism Auth API retuned an error code: {} for the call to {}.  Maybe an ENTERPRISE feature?'.format( myruntime, e, reqstring)
         data = "ERROR"
         sys.exit(0)
-
+  
     return data['access_token']
+    
+def getAccessToken(username,password,myApiKey):
```

Comment:
Why are we reverting this change though?  Is it the breaking change?  Please test and comment if so.